expressions.js

```diff
@@ -88,7 +90,19 @@ module.exports = {
 
   oclc: {
     expr: /\/search\/o\=?(\d+)/,
-    handler: match => `${BASE_SCC_URL}/search?oclc=${match[1]}&redirectOnMatch=true`,
+    handler: async (match) => {
+      // check if bib is research or circulating
+      const oclcNum = match[1]
+      const resp = await nyplApiClient({ apiName: 'discovery' })
+        .then(client =>
```

Comment:
Readability nitpick: Given the function is already `async`, this could be structured as two serial `awaits` - the first for the `client` and the second for the `resp`

Reply:
Fixed